templates/welcome.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Cookie Example</title>
+  </head>
+  <body>
+    <h1>Добро пожаловать, {{ name }}!</h1>
+    <form method="post" action="/logout">
```

Comment:
Но тут нужно убрать `action`,  иначе работать не будет
```suggestion
    <form method="post">
```